fs/localfs/local_fs_test.go

```diff
@@ -22,6 +22,50 @@ type fileEnt struct {
 	isFile bool
 }
 
+func TestSymlink(t *testing.T) {
+	tmp := testutil.TempDirectory(t)
+
+	fn := filepath.Join(tmp, "target")
+	absLink := filepath.Join(tmp, "abslink")
+	relLink := filepath.Join(tmp, "rellink")
+
+	assertNoError(t, os.WriteFile(fn, []byte{1, 2, 3}, 0o777))
+	assertNoError(t, os.Symlink(fn, absLink))
+	assertNoError(t, os.Symlink("./target", relLink))
+
+	verifyLink(t, absLink, fn)
+	verifyLink(t, relLink, fn)
+}
+
+func verifyLink(t *testing.T, path, expected string) {
+	t.Helper()
+
+	ctx := testlogging.Context(t)
+
+	entry, err := NewEntry(path)
+	require.NoError(t, err)
+
+	if link, ok := entry.(fs.Symlink); !ok {
+		t.Errorf("entry is not a symlink: %s", path)
```

Comment:
`require.True(t, ok, "entry is not a symlink: %s", path)`

Reply:
Ditto, can't find this